proxy/http/HttpTransactCache.cc

```diff
@@ -1325,28 +1283,21 @@ HttpTransactCache::match_response_to_request_conditionals(HTTPHdr *request, HTTP
 
   // If-None-Match: may match weakly //
   if (request->presence(MIME_PRESENCE_IF_NONE_MATCH)) {
-    int raw_etags_len, comma_sep_tag_list_len;
-    const char *raw_etags          = response->value_get(MIME_FIELD_ETAG, MIME_LEN_ETAG, &raw_etags_len);
-    const char *comma_sep_tag_list = nullptr;
-
-    if (raw_etags) {
-      comma_sep_tag_list = request->value_get(MIME_FIELD_IF_NONE_MATCH, MIME_LEN_IF_NONE_MATCH, &comma_sep_tag_list_len);
-      if (!comma_sep_tag_list) {
-        comma_sep_tag_list     = "";
-        comma_sep_tag_list_len = 0;
-      }
+    TextView raw_etag = response->value_get(TextView{MIME_FIELD_ETAG, MIME_LEN_ETAG});
```

Comment:
It is consistent - at this code site, it's all `TextView`. There's no detectable performance difference between using `string_view` vs `TextView` for the argument passing. If you want a real fix for this, I would add things like `MIME_VIEW_ETAG` and use that instead of an explicit type. I can set up a PR for that if you like.

Reply:
I just removed the references to `TextView` entirely. Problem solved.